content/joysticks/winwing/winwing-cdu/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: WINWING CDU
+weight: 50
+---
+
+WINWING CDU display support is automatic on run, if a supported plane and a CDU are detected. To prevent MobiFlight from taking over the CDU, disable it in `Extras->Settings->Peripherals`.
+
+## Installing Python
+
+Follow the [guide](/guides/installing-python/).
+
+## General remarks
+
+- Python is a mandatory prerequisite.
```

Comment:
I think the header and the python note can be removed to simplify the page if the above important block change is made.